app/src/main/java/ie/setu/mad2_assignment_one/ui/shopping/ShoppingListScreen.kt

```diff
@@ -45,6 +48,8 @@ import kotlinx.coroutines.tasks.await
 fun ShoppingListScreen(modifier: Modifier = Modifier, onNavigateBack: () -> Unit, shoppingListViewModel: ShoppingListViewModel, onItemClick: (ShoppingItem) -> Unit) {
     val scope = rememberCoroutineScope()
     val storage = Firebase.storage
+    // modal boolean
+    val showModal = remember { mutableStateOf(false) }
```

Comment:
`showModal` is `displayModal` - replace in description to reflect true variable name.